vault/resource_approle_auth_backend_role_secret_id_test.go

```diff
@@ -57,6 +58,30 @@ func TestAccAppRoleAuthBackendRoleSecretID_wrapped(t *testing.T) {
 	})
 }
 
+func TestAccAppRoleAuthBackendRoleSecretID_wrapped_reuse(t *testing.T) {
+	backend := acctest.RandomWithPrefix("approle")
+	role := acctest.RandomWithPrefix("test-role")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { util.TestAccPreCheck(t) },
```

Comment:
`TestAccPreCheck()` was moved to the `testutil` package.

```suggestion
		PreCheck:     func() { testutil.TestAccPreCheck(t) },
```